### ISSUE TITLE
Not change position of UITabBar while Reveal is appear.

### ISSUE THREAD

**Issue by Pramod-ibl** [John-Lluch/SWRevealViewController#536]
I implement SWRevealViewController with UITabbarController via storyboard. I want to display SWRevealViewController in first UITabBarItem only, in other UITabBarItem i don't want to open SWRevealViewController, that done completely. **But problem is position of UITabBar is not change like viewcontroller while SWRevealViewController is appear.**

Please help me.
**Output**
![Output](https://cloud.githubusercontent.com/assets/16754283/12410011/834ed364-be96-11e5-93db-4268553c3b51.png)

**Storyboard Structure**
![Storyboard Structure](https://cloud.githubusercontent.com/assets/16754283/12410006/5bd96ed4-be96-11e5-9a55-a4d0ce5c10ec.png)
)

**Comment by iDevelopper** [comment]
Why did you add a navigation controller between tab bar controller and reveal view controller? You don't have to do this. Or did you want to say position of **Nav Bar**? If it is, you have to add a navigation controller between Reveal View Controller and Table View Controller (i.e the menu).

![2016-01-19_07-53-46](https://cloud.githubusercontent.com/assets/1782450/12411863/2cddf550-be82-11e5-81f2-95fd2de86300.png)

![2016-01-19_08-02-00](https://cloud.githubusercontent.com/assets/1782450/12411962/1608df10-be83-11e5-810b-c1bb345968d9.png)

**Comment by Pramod-ibl** [comment]
@iDevelopper i remove navigation controller between UITabbarController and SWRevealViewController than also not change position of UITabbar. NavigationBar work completely I want to change position of UITabbar also.

**Comment by iDevelopper** [comment]
Then I don't understand the question. Where do you want the tab bar to appear. A tab bar is always at the bottom off the screen.

**Comment by Pramod-ibl** [comment]
@iDevelopper See my output image in question. In that image position of navigation bar is change while reveal is come(go to right), but not change position of tabbar(go to right). i want to change position of tabbar(go to right) while reveal while appear like navigationbar. i want output like following.

![output](https://cloud.githubusercontent.com/assets/16754283/12413644/7f8fb868-beb6-11e5-9491-000d69302cdb.png)

**Comment by iDevelopper** [comment]
The tab bar controller is the root view controller of the application and it is the entry point. Then you have to change the frame of it's tab bar programmatically each time the front view position change. See the delegate methods of SWRevealViewController. When rear view is visible the x position of the tabBar will be revealViewController.rearViewRevealWidth

**Comment by Pramod-ibl** [comment]
@iDevelopper  any other way ??

**Comment by iDevelopper** [comment]
No, but in fact I don't understand why you want your tab bar move with the front view? When the user want to tap another tab item, he can't do it when the menu is open !! If you want really do that, the solution above is very simple.

**Comment by iDevelopper** [comment]
https://www.dropbox.com/sh/v774x16bq9u6gvn/AADUv0ezOw6AljAG0LupHAfXa?dl=0

**Comment by Pramod-ibl** [comment]
@iDevelopper  Thank you so much...

**Comment by iDevelopper** [comment]
I would like to know why you want to do this, please

**Comment by Pramod-ibl** [comment]
@iDevelopper  I think not looking good if not change position of UITabbar.
